resources/packs/k8s/common.go

```diff
@@ -241,7 +247,8 @@ func initNamespacedResource[T K8sNamespacedObject](
 		}
 	}
 
-	return args, *new(T), fmt.Errorf("not found")
+	// the error K8sObjectNotFound is checked by cnspec
+	return args, *new(T), &K8sObjectNotFound{}
```

Comment:
Where will the error be checked? It seems like in `runtime.go` you are not returning that error type anymore but just `error`. Is it intentional?

Reply:
It is check in cnspec: https://github.com/mondoohq/cnspec/pull/131/files#diff-0a279e72affdf3ff0b6e366176e86400378905934fa1b2203a50a9cae6ae74cbR323